agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/Feature.java

```diff
@@ -60,7 +60,8 @@ enum Feature {
   VERTX_DISABLED(29), // preview instrumentation, vertx is ON by default in OTEL
   CUSTOM_DIMENSIONS_ENABLED(30), // enable customDimensions
   JAXRS_ANNOTATIONS_DISABLED(
-      31); // can cause startup slowness, jaxrs-annotations is ON by default in OTEL
+      31), // can cause startup slowness, jaxrs-annotations is ON by default in OTEL
+  LOGGING_LEVEL_ENABLED(32); // preview opt-in to include LoggingLevel.
```

Comment:
`LOGGING_LEVEL_CUSTOM_PROPERTY_ENABLED` is longer, but I think worth being explicit since we already have confusion between instrumentation logging level and self-diagnostics logging level